calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The Tigera operator is deployed as a ReplicaSet in the `tigera-operator` namespace, and records status in a custom resource named, `tigerastatus`. The operator get its configuration from several Custom Resources (CRs); the central one is the Installation CR.
```

Comment:
ReplicaSet -> Deployment

Reply:
I think we should call these "API resources" instead of custom resources. The end user doesn't need to know they are implemented using CRDs, just that it's an API available via the Kubernetes API server. 

e.g.,

> The operator gets its configuration from several Kubernetes API resources, and records status using TigeraStatus Kubernetes API resources in the operator.tigera.io API group.

Reply:
Fixed.